index.bs

```diff
@@ -2109,8 +2109,8 @@ an |ownership type|, a |serialization internal map|, a |realm| and a |session|:
 
                1. Let |serialized shadow| be the result of
                   [=serialize as a remote value=] with |shadow root|, |max depth|,
-                  false, |ownership type|, |serialization internal map|,
```

Comment:
Looks like `false` is not needed here, since [serialize as a remote value](https://w3c.github.io/webdriver-bidi/#serialize-as-a-remote-value) accepts only 6 arguments which match all the other mentioned here.